vms/platformvm/signer/proof_of_possession.go

```diff
@@ -43,7 +43,7 @@ func NewProofOfPossession(sk *bls.SecretKey) *ProofOfPossession {
 }
 
 func (p *ProofOfPossession) Verify() error {
-	publicKey, err := bls.PublicKeyFromBytes(p.PublicKey[:])
+	publicKey, err := bls.DecompressPublicKey(p.PublicKey[:])
```

Comment:
This no longer correctly validates the public key - which would open up a terrifying series of vulns

Reply:
Will verify all paths and add some tests since the core looks reasonable but yeah..